test.js

```diff
@@ -1,36 +1,59 @@
 'use strict';
-var assert = require('assert');
+var assert = require('chai').assert;
```

Comment:
@sindresorhus is the outdated mocha suggestion to me in regards to lacking useful output on failed assertions? I am using whichever version is coming with npm install for the package. In my case, `2.2.5` (which is the latest mocha release).